serverless.yml

```diff
@@ -66,7 +67,7 @@ functions:
     handler: todos/delete.delete
     events:
       - http:
-          path: todos/{id}
+          path: todos/{id}/{userId}
```

Comment:
Since for now we should only be able to see our own notes, we could get this userId from event that lambda function receives.